packages/wrangler/src/init.ts

```diff
@@ -461,8 +470,7 @@ export async function initHandler(args: ArgumentsCamelCase<InitArgs>) {
 			);
 			if (fromDashScriptName) {
 				logger.warn(
-					`After running "wrangler init --from-dash", modifying your worker via the Cloudflare dashboard is discouraged.
-					Edits made via the Dashboard will not be synchronized locally and will be overridden by your local code and config when you publish.`
+					`After running "wrangler init --from-dash", modifying your worker via the Cloudflare dashboard is discouraged.\nEdits made via the Dashboard will not be synchronized locally and will be overridden by your local code and config when you publish.`
```

Comment:
Same comment as above re chalk and `wrangler init --from-dash` (or at least consistency between " and `)